sources/packages/api/src/services/application/application.service.ts

```diff
@@ -132,13 +151,17 @@ export class ApplicationService extends RecordDataModelService<Application> {
       application.applicationStatus = ApplicationStatus.submitted;
       application.relationshipStatus = applicationData.relationshipStatus;
       application.studentNumber = applicationData.studentNumber;
-      application.applicationStatusUpdatedOn = getUTCNow();
+      application.applicationStatusUpdatedOn = now;
       application.studentFiles = await this.getSyncedApplicationFiles(
         studentId,
         application.studentFiles,
         associatedFiles,
       );
-      return this.repo.save(application);
+      application.modifier = auditUser;
+      application.studentAssessment = [originalAssessment];
+
+      await this.repo.save(application);
+      return { application, assessment: originalAssessment };
```

Comment:
when I saw the code where submitApplication is called, 

I saw const {assessment} = this.service.submitApplication() which made me think that the property assessment from Application(Entity) is being parsed here.

Hence in line: 164 can we use a property name which is not existing in application entity. like studentAssessment.?

Reply:
I believe that the below code syntax is pretty clear about destructuring the result of the method.
I would like a second option on this: @ann-aot @guru-aot @henryoforeh-aot 
```
const { assessment } = this.service.submitApplication();
```

Reply:
Renamed _assessment_ to _createdAssessment_ as we talked.